### PR TITLE
Make very minor stylistic suggestions to the paper.

### PR DESCRIPTION
**Pull request by drvinceknight**
Some very minor stylistic/language suggestions (as part of https://github.com/openjournals/joss-reviews/issues/1371).

### CODE REVIEW

File: paper/paper.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ bibliography: paper.bib
 
 # Summary
 
-There has been a remarkable effort in the last years in
+There has been a remarkable effort in recent years in
 the robotics community to formulate estimation problems
 properly. This is motivated by an increasing demand for
```
Comment (drvinceknight) [inline paper/paper.md]: Would it be helpful to a general reader to add either a small (1 sentence) description of the problems OR perhaps point at the mathematical paper for more details? As I'm not familiar with this area I don't really understand what the package is for.

**Comment by joansola** [comment]
Hi Vince. Thanks for this input. I am making some changes in the intro of the library. Regarding the paper, I'll also consider a better introduction, though this will be handled later.

**Comment by artivis** [comment]
@drvinceknight your remarks were addressed in #81. If you agree, I'll close this PR.

**Comment by drvinceknight** [comment]
Thanks, looks good to me :+1:

**Pull request closed without merging**